source/client/options_impl.cc

```diff
@@ -169,6 +169,18 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "", "trace", "Trace uri. Example: zipkin://localhost:9411/api/v1/spans. Default is empty.",
       false, "", "uri format", cmd);
 
+  std::vector<std::string> h1_connection_reuse_strategies = {"HOT", "FAIR"};
+  TCLAP::ValuesConstraint<std::string> h1_connection_reuse_strategies_allowed(
+      h1_connection_reuse_strategies);
+  TCLAP::ValueArg<std::string> h1_connection_reuse_strategy(
+      "", "h1-connection-reuse-strategy",
+      fmt::format(
+          "Choose picking the most recently used, or least-recently-used connections for re-use."
+          "(default: {}).",
+          absl::AsciiStrToLower(
+              nighthawk::client::H1ConnectionReuseStrategy_H1ConnectionReuseStrategyOptions_Name(
+                  h1_connection_reuse_strategy_))),
+      false, "", &h1_connection_reuse_strategies_allowed, cmd);
```

Comment:
Shouldn't we set and document the default as HOT, as if I understand this correctly, that is what happens when the flag isn't specified?

Reply:
It will be documented here, as that will replace {} with the default: https://github.com/envoyproxy/nighthawk/pull/198/files/59cefde2ee9417889f98a5af69a5e968e9cc3bb5#diff-54af86e901ab4e3f97d4dcf19e326cc3R178

The default is defined here in the options header (all trivially typed options have their defaults set over there): https://github.com/envoyproxy/nighthawk/pull/198/files/59cefde2ee9417889f98a5af69a5e968e9cc3bb5#diff-139219ef3d84f04ddb20aef0d38419dfR89